httphandlers/buildinfo_handler.go

```diff
@@ -3,12 +3,21 @@ package httphandlers
 import (
 	"encoding/json"
 	"github.com/Financial-Times/service-status-go/buildinfo"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
 	"net/http"
 )
 
-// BuildInfoHandler uses the buildinfo.GetBuildInfo to respond to requests for build info !
-func BuildInfoHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
We were just about to use it hence the question about renaming it. If we use it now then this release will break 10 projects :)